src/connectdlg.cpp

```diff
@@ -220,15 +211,17 @@ void CConnectDlg::RequestServerList()
 
     // update list combo box (disable events to avoid a signal)
     cbxCentServAddrType->blockSignals ( true );
-    cbxCentServAddrType->setCurrentIndex ( static_cast<int> ( pSettings->eCentralServerAddressType ) );
+    // iCustomDirectoryIndex is non-zero only if eCentralServerAddressType == AT_CUSTOM
```

Comment:
"... and is thus the offset into cbxCentServAddrType after the last non-custom directory server"

Reply:
hmm the combo box name doesn't really reflect its true function anymore.

Reply:
`cbxDirectoryServer` would do.  Feel free :)